xbmc/pvr/PVRPlaybackState.cpp

```diff
@@ -367,6 +370,7 @@ void CPVRPlaybackState::StartPlayback(CFileItem* item,
     }
     else if (item->IsEPG())
     {
+      client->StreamClosed();
```

Comment:
add error handling

Reply:
Added a log statement in the event the clint returns an error. Is that sufficient?